CsharpExtras/Map/Dictionary/Curry/ICurryDictionary.cs

```diff
@@ -0,0 +1,66 @@
+﻿using CsharpExtras.ValidatedType.Numeric.Integer;
+using System.Collections.Generic;
+
+namespace CsharpExtras.Map.Dictionary.Curry
+{
+    public interface ICurryDictionary<TKey, TVal>
+    {
+        NonnegativeInteger Arity { get; }
+        TVal this[params TKey[] keyTuple] { get; set; }
```

Comment:
Is there a reason to have a `set` property here? Neither of the implementations seem to implement it.

Reply:
Yeah that ``set`` shouldn't be there. Good catch.